env0/resource_cost_credentials.go

```diff
@@ -0,0 +1,208 @@
+package env0
+
+import (
+	"context"
+	"errors"
+
+	"github.com/env0/terraform-provider-env0/client"
+	. "github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceCostCredentials() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceCostCredentialsCreate,
+		ReadContext:   resourceCostCredentialsRead,
+		DeleteContext: resourceCostCredentialsDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Description: "name for the credentials",
+				Required:    true,
+				ForceNew:    true,
+			},
+			"arn": {
+				Type:          schema.TypeString,
+				Description:   "the aws role arn",
+				Optional:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"client_id", "client_secret", "tenant_id", "subscription_id", "table_id", "secret"},
```

Comment:
Indentations add space for all the `ConflictsWith` so it will be aligned.
Currently it's like:
  bla
  bla
 []string{"client....   <---------
  bla